modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts

```diff
@@ -178,3 +178,40 @@ test('LPD-37727 Change delta to a higher value when on last page', async ({
 		page.getByRole('link', {name: /Thread with headline #\d+/})
 	).toHaveCount(5);
 });
+
+test('LPD-39570 Change delta to a higher value when on last page', async ({
+	apiHelpers,
+	messageBoardsWidgetPage,
+	page,
+	site,
+}) => {
+	for (let i = 0; i < 5; i++) {
+		await apiHelpers.headlessDelivery.postMessageBoardThread({
+			articleBody: getRandomString(),
+			headline: 'Thread with headline #' + i,
+			siteId: site.id,
+		});
+	}
+
+	await messageBoardsWidgetPage.addMessageBoardsPortlet(site);
+
+	await page.getByLabel('Items per Page').click();
+	await page.getByRole('option', {name: /4\s+Entries per Page/}).click();
```

Comment:
I think this is a side effect of the copy paste or somthing like that, all this "/", "\\s" we don't need them, we can just put: `{name: '4 Entries per Page'}`

Reply:
This is intentional. The problem is that the label is "4_Entries per Page", where the '_' may be a mix of ASCII spaces (0x20) and &nbsp; (in UTF-8 0xC0 0xA0).